tests/e2e/chain/export.go

```diff
@@ -8,16 +8,21 @@ type ChainMeta struct {
 }
 
 type Validator struct {
-	Name          string `json:"name"`
-	ConfigDir     string `json:"configDir"`
-	Index         int    `json:"index"`
-	Mnemonic      string `json:"mnemonic"`
-	PublicAddress string `json:"publicAddress"`
+	Name           string `json:"name"`
+	ConfigDir      string `json:"configDir"`
+	Index          int    `json:"index"`
+	Mnemonic       string `json:"mnemonic"`
+	PublicAddress  string `json:"publicAddress"`
+	PublicAddress2 string `json:"publicAddress2"`
+	PublicKey      string `json:"publicKey"`
+	OperAddress    string `json:"operAddress"`
 }
 
 type Chain struct {
 	ChainMeta  ChainMeta    `json:"chainMeta"`
 	Validators []*Validator `json:"validators"`
+	PropNumber int          `json:"propNumber"`
+	LockNumber int          `json:"lockNumber"`
```

Comment:
Just checked that both `PropNumber` and `LockNumber` are currently not used during initialization. So they should be in the test suite instead. These structs are primarily for passing chain initialization data between docker `chain_init` container and the test runner